bcipy/simulator/task/null_display.py

```diff
@@ -0,0 +1,18 @@
+"""Display that doesn't do anything"""
+from typing import List, Tuple
+
+from bcipy.display import Display
+
+
+class NullDisplay(Display):
```

Comment:
This could live in the display module or the helpers/display; I could see this being useful for other headless operations!

Reply:
There are a number of classes and functions that I think may be helpful in other parts of the BciPy codebase, but I wanted to keep this PR relatively clean to avoid merge conflicts with the orchestrator work. I created a ticket to track this followup: https://www.pivotaltracker.com/story/show/188328498